LobbyServer2/BridgeServer/BridgeServerProtocol.cs

```diff
@@ -509,13 +537,22 @@ protected override async void HandleMessage(MessageEventArgs e)
                 log.Debug($"< {request.GetType().Name} {DefaultJsonSerializer.Serialize(request)}");
                 log.Info($"Game {GameInfo?.Name} {request.GameStatus}");
 
-                UpdateGameStatus(request.GameStatus, true);
+                ServerGameStatus = request.GameStatus;
 
-                if (GameStatus == GameStatus.Stopped)
+                if (ServerGameStatus == GameStatus.Stopped)
                 {
-                    foreach (LobbyServerProtocol client in clients)
+                    foreach (LobbyServerProtocol client in GetClients())
                     {
                         client.CurrentServer = null;
+
+                        //Unready people when game is finishd
+                        ForceMatchmakingQueueNotification forceMatchmakingQueueNotification = new ForceMatchmakingQueueNotification()
+                        {
+                            Action = ForceMatchmakingQueueNotification.ActionType.Leave,
+                            GameType = GameType.PvP
```

Comment:
Probably we should pull current gametype

Reply:
yeah we can fix it
```                        
if (GameInfo != null)
{ 
    //Unready people when game is finisht
    ForceMatchmakingQueueNotification forceMatchmakingQueueNotification = new ForceMatchmakingQueueNotification()
    {
        Action = ForceMatchmakingQueueNotification.ActionType.Leave,
        GameType = GameInfo.GameConfig.GameType
    };
    client.Send(forceMatchmakingQueueNotification);
}
```
                        
but to make pr just for this :) so can be added to next pr we do by someone

Reply:
why not

Reply:
#54 done